app/views/admin/posts/_form.html.haml

```diff
@@ -1,5 +1,8 @@
 = effective_form_with(model: post, url: post.persisted? ? effective_posts.admin_post_path(post.id) : effective_posts.admin_posts_path) do |f|
   = f.text_field :title, hint: 'The title of your post.'
```

Comment:
Could you also add this same stuff to the form at 'app/views/effective/posts/_form`

We have an admin and a front facing form. Both can use the slugs I guess :)

Reply:
Oh cool! Added.